### PR TITLE
initial draft

### PR DESCRIPTION
**Pull request by storopoli**

### CODE REVIEW

File: 01-single_tte.jl [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        # the derivative of cumulative hazard is equal to hazard
+        Λ' = λ
+    end
+
+    @derived begin
+        DV ~ @. TimeToEvent(λ, Λ) # special type of distribution
+    end
+end
+
+tte_single_weibull_fit = fit(
```
Comment (jotas6) [inline 01-single_tte.jl:102]: I am not sure if this is relevant, but I am getting these warnings here:

weibull

It then continues running and at the end it says that the minimization was successful.

File: 01-single_tte.jl [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        # the derivative of cumulative hazard is equal to hazard
+        Λ' = λ
+    end
+
+    @derived begin
+        DV ~ @. TimeToEvent(λ, Λ) # special type of distribution
+    end
+end
+
+tte_single_gompertz_fit = fit(
```
Comment (jotas6) [inline 01-single_tte.jl:145]: I am also getting the instability warnings here, but in the end the minimization is successful.

gompertz

File: docs/instructors.md [hunk 3]
```diff
@@ -1,10 +1,55 @@
...
+For the Weibull model the parameters are almost the same as the exponential model,
+but with the addition of the shape parameter `Κ`.
+In the `@vars` block we use the Weibull formula for the hazard function `λ`.
+The Gompertz model follows a similar pattern as the Weibull model.
+
+In all of these three models, we are using the special distribution `TimeToEvent`.
+For the fitting procedure we cannot use `FOCE()` and we need to use either `NaivePooled()` or `LaplaceI()`.
+
+Finally, you can compare the estimates with the `compare_estimates` from `PumasUtilities`.
+If you find opportune, you can go over model comparisons between the three models with fit metrics, e.g. AIC,
```
Comment (jotas6) [inline docs/instructors.md:50]:
```suggestion
If you find it opportune, you can go over model comparisons between the three models with fit metrics, e.g. AIC,
```

**Review by jotas6 — COMMENTED** [review]
Very small comments

File: 01-single_tte.jl (hunk 1 above)
Comment (storopoli) [reply 01-single_tte.jl:102]: Yes, it is not a problem.

File: 01-single_tte.jl (hunk 2 above)
Comment (storopoli) [reply 01-single_tte.jl:145]: Not a problem.

**Review by jotas6 — APPROVED** [review]

**Pull request merged**